File: tarea/ejercicio08.java
import java.util.Scanner;
class ejercicio08{
static Scanner teclado=new Scanner(System.in);
static void ejercicio08(){
  //definir variables
  int anhoant;
  double sueldo, bonoant=0, bonosueld=0, bonoreal=0;
  //datos de entrada
  System.out.print("Ingrese los años de antiguedad:");
  anhoant=teclado.nextInt();
  System.out.print("Ingrese el sueldo del trabajador:");
  sueldo=teclado.nextDouble();
  //proceso
  if(anhoant>2 && anhoant<5){
    bonoant=sueldo*0.20;
  }else if(anhoant<=5){
    bonoant=sueldo*0.30;
  }
  if(sueldo<1000){
    bonosueld=sueldo*0.25;
  }else if(sueldo<=1000 && sueldo<=3500){
    bonosueld=sueldo*0.15;
  }else{
    bonosueld=sueldo*0.10;
  }
  if(bonoant>=bonosueld){
    bonoreal=bonoant;
  }else{
    bonoreal=bonosueld;
  }
  //datos de salida
  System.out.println("El trabajador tendrá un bono de:" +bonoreal+ "\nSin embargo su bono por antiguedad es: "+bonoant+ "\n y el bono sueldo es:"+bonosueld);
  
  }
  public static void main(String[] arg){
  ejercicio08();
}
}

File: tarea/ejercicio08.py
def ejercicio08():
  print ("Sueldo y bono para el empleado")
  sueldo=int(input("Ingrese el sueldo actual que recibe:"))
  tiempo=int(input("Ingrese el tiempo que lleva trabajando en la empresa:"))
   #Proceso
  sueldot=0.0
  sueldo1=0.0
  sueldo2=0.0
  if tiempo>2 and tiempo<5:
    sueldo1=(sueldo*0.2)
  else:
    sueldo1=(sueldo*0.3)

  if sueldo<1000:
    sueldo2=(sueldo*0.25)
  elif sueldo>1000 and sueldo<=3500:
    sueldo2=(sueldo*0.15)
  elif sueldo>3500:
    sueldo2=(sueldo*0.1)
  
  if sueldo1>sueldo2:
    sueldot=sueldo1
  elif sueldo2>sueldo1:
    sueldot=sueldo2
    
  
  print("El bono que tendrá será de:", sueldot)
ejercicio08()
